rdt/hyper_transformer.py

```diff
@@ -78,6 +78,7 @@ class HyperTransformer:
         'O': 'categorical',
         'b': 'boolean',
         'M': 'datetime',
+        'p': 'pii',
```

Comment:
This is used to pick the `RDT` data type based on a pandas `dtype`. I don't think they have one that is `p` so this won't do anything. We can get rid of this line